thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/template.hbs

```diff
@@ -285,19 +287,13 @@
     </div>
 
   {{else}}
-
     <div class="te-alert-page-pending">
-      <img src="{{rootURL}}assets/images/te-alert-{{if isReplayStatusError "error" "pending"}}.png" class="te-alert-page-pending__image {{if isReplayStatusError "te-alert-page-pending__image--error"}}" alt="alertData.Setup is Processing">
-      <h2 class="te-alert-page-pending__title">{{if isReplayStatusError "Oops, something went wrong." "Setting up your alert"}}</h2>
-      {{#if (not isReplayStatusError)}}<div class="te-alert-page-pending__loader"></div>{{/if}}
+      <img src="{{rootURL}}assets/images/te-alert-pending.png" class="te-alert-page-pending__image" alt="alert setup processing">
```

Comment:
Did {{rootURL}} work?

Reply:
I believe it was working all along, I just had some complex interfering boolean conditionals blocking it.